scripts/installation/install_solc.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import platform
+import shutil
+from os.path import dirname, join, abspath
+from os import stat, chmod
+import requests
+
+
+if platform.system() != 'Linux':
+    raise EnvironmentError("This installation script is only compatible with linux-gnu-based operating systems.")
+
+PACKAGE_NAME = join('nucypher', 'blockchain', 'eth', 'sol')
+BASE_DIR = dirname(dirname(dirname(abspath(__file__))))
+FILE_PATH = join(BASE_DIR, PACKAGE_NAME, "__init__.py")
+
+METADATA = dict()
+with open(FILE_PATH) as f:
+    exec(f.read(), METADATA)
+
+solc_version = METADATA['SOLIDITY_COMPILER_VERSION']
+
+# Get solc binary for linux
+solc_bin_path=join(dirname(shutil.which('python')), 'solc')
```

Comment:
Some convention comments: Use the same case for names in the same scope - `SOLC_BIN_PATH` for example. Perhaps make a function to encapsulate the install functionality (`def main`) and include a call at the bottom of this file:
```
if __name__ == "__main__":
    main()
```

Reply:
✔️